aries_cloudagent/revocation/routes.py

```diff
@@ -157,11 +157,16 @@ def validate_fields(self, data, **kwargs):
 
         notify = data.get("notify")
         connection_id = data.get("connection_id")
+        notify_version = data.get("notify_version")
 
         if notify and not connection_id:
             raise ValidationError(
                 "Request must specify connection_id if notify is true"
             )
+        if notify and not notify_version:
```

Comment:
Can we default to `v1` notifications if not specified?  That way it's not a breaking change.

Reply:
Yep! I took the word "required" and assumed that it was supposed to be a breaking change. I'll set it up to default to v1.

Reply:
I have left the check in and added a default value to the get lines. Verified that this works by removing the argument from the behavior tests